index.json

```diff
@@ -321,13 +321,13 @@
     "128px"
   ],
   "fontFamily": {
-    "arialBlack": "'Arial Black', 'Arial Bold', Gadget, sans-serif",
-    "lato": "Lato, -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Noto Sans', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'",
-    "mono": "'Roboto Mono', ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace",
-    "monoSystem": "ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace",
-    "sansSystem": "-apple-system, BlinkMacSystemFont, 'Segoe UI', 'Noto Sans', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'",
-    "serifSystem": "Georgia, serif"
-  },
+		"arialBlack": "'Arial Black', 'Arial Bold', Gadget, sans-serif",
+		"lato": "Lato, -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Noto Sans', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'",
+		"mono": "'Roboto Mono', ui-monospace, SFMono-Regular, 'SF Mono', Menlo, Consolas, monospace",
```

Comment:
Context note on adding single quotes around `SF Mono` - these are a good gate for any string that contains more than one word in CSS.